src/Symfony/Component/Translation/Translator.php

```diff
@@ -158,17 +157,16 @@ public function setFallbackLocales(array $locales)
             $this->assertValidLocale($locale);
```

Comment:
It would be nice if the `FallbackLocaleProvider` could use this guard itself, but I am unsure where to move the code.